com.hybris.hyeclipse.utils/src/com/hybris/hyeclipse/testers/AbstractFilePropertyTester.java

```diff
@@ -48,13 +47,20 @@ protected boolean testSelectedFileByExtensions(final Object receiver, final Set<
 				return fileExtensions.contains(file.getFileExtension());
 			}
 		} else if (receiver instanceof List) {
+			boolean doesExtensionMatch = false;
```

Comment:
I do, java doc has been updated and code refactored to single return. I think lambda is not good choice for this scenario.